call/rtp_transport_controller_send.cc

```diff
@@ -556,8 +558,12 @@ void RtpTransportControllerSend::OnTransportFeedback(
                                                              feedback_time);
     if (feedback_msg) {
       if (controller_)
-        PostUpdates(controller_->OnTransportPacketsFeedback(*feedback_msg));
-
+#ifdef INTEL_GPRA
```

Comment:
We will not enable GPRA any more. Please remove this.

Reply:
It looks like some INTEL_GPRA and AllowExternalBwe is not really for GPRA, but some general optimization for BWE. Would you prefer to remove all of them in this PR or remove them in a separate PR for easy review?

Reply:
Fine with a separate PR for removal.